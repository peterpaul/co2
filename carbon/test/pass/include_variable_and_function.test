include inc/TypeDefB
include inc/TypeDefA

["stdlib.h"] {
	int printf (char *msg, ...);
}
int main () {
	foo = 5;
	printf ("bar (8) = %d\n", bar (8));
	return 0;
}
