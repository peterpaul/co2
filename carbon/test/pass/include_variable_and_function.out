bar (8) = 13
